### PR TITLE
feat: rule used to generate cnv report file

### PR DESCRIPTION
**Pull request by Smeds**
### Review and tests:
- [ ] Tests pass
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Code review
- [ ] `CHANGELOG.md` is updated
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

**Comment by jonca79** [comment]
There is no .env file for conda.

File: workflow/rules/cnv_tsv_report.smk [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        mem_per_cpu=config.get("cnv_tsv_report", {}).get("mem_per_cpu", config["default_resources"]["mem_per_cpu"]),
+        partition=config.get("cnv_tsv_report", {}).get("partition", config["default_resources"]["partition"]),
+        threads=config.get("cnv_tsv_report", {}).get("threads", config["default_resources"]["threads"]),
+        time=config.get("cnv_tsv_report", {}).get("time", config["default_resources"]["time"]),
+    container:
+        config.get("cnv_tsv_report", {}).get("container", config["default_container"])
+    conda:
+        "../envs/cnv_tsv_report.yaml"
+    message:
+        "{rule}: Convert cnv vcf to a tsv file: {output.result}"
```
Comment (jonca79) [inline workflow/rules/cnv_tsv_report.smk:31]:
```suggestion
        "{rule}: Convert cnv vcf to a tsv file: {output.tsv}"
```

Comment (Smeds) [reply workflow/rules/cnv_tsv_report.smk:31]: :+1:

**Comment by Smeds** [comment]
> There is no .env file for conda.

:facepalm: fixed

**Review by jonca79 — APPROVED** [review]

**Pull request merged**